instruments/iama.yaml

```diff
@@ -26,7 +26,7 @@ template: &assessmentcard_template                  # anchor for all fields belo
 
 tasks: # each question can be seen as a separate task
   - question: "Licht uw voorstel voor het gebruik/de inzet van een algoritme toe. Wat is de aanleiding hiervoor geweest? Voor welk probleem moet het algoritme een oplossing bieden?"
-    urn: "urn:nl:aivt:ir:iama:1.0:1.1.1"
+    urn: "urn:nl:aivt:ir:iama:q:1.0:1.1.1"
```

Comment:
why is this `q` introduced?

Reply:
The q is introduced as the type of task is a question, however as @laurensWe suggested in another comment: would make sense to group them in task types instead of in the urn itself.